script/GameObjects/Elements/Components/Capacitor.lua

```diff
@@ -17,7 +17,7 @@ local Bindings    = require('States.ApplicationBindings')
 local shared
 local rng         = RNG.FromTime()
 
-local Capacitor   = Subclass(Entity, function(self)
+local Capacitor   = Subclass("", Entity, function(self)
```

Comment:
Looks like some names weren't copied over due to the regular expression used not checking for empty spaces. I think it's safe to ignore for now since the empty string name shouldn't produce any errors.

Reply:
Well spotted, thanks. I manually fixed these as there were only 6 of them.